src/azure-cli/azure/cli/command_modules/netappfiles/_breaking_change.py

```diff
@@ -0,0 +1,16 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from azure.cli.core.breaking_change import register_argument_deprecate
+
+register_argument_deprecate('netappfiles volume create', '--is-restoring', target_version='2.73.0')
```

Comment:
The next release is 2.73.0, actually you should create PR for breaking change now. But I will merge it first so that our breaking change notes will include those items

Reply:
We need this in:   2.72.0   05/06/2025

Reply:
@kairu-ms thoughts on above can we release this in 2.72.0,  05/06/2025?

Reply:
The code freeze day is 2 weeks ago

Reply:
Usually those breaking change warnings messages should be merged at least  2 versions before breaking changes. Such as 2.71.0

Reply:
We started this pr April 11, but there was little tracktion on it, what would be the best way to get things reviewd so we get things reviewed in time?

@kairu-ms  good to know about the breaking changes, we will keep that in mind thanks.

One question at what point does this get updated  ?
https://learn.microsoft.com/en-us/cli/azure/upcoming-breaking-changes